.github/workflows/pr_push.yml

```diff
@@ -141,7 +141,10 @@ jobs:
       run: cmake --build build --target clang-format-check
 
     - name: Check CMake formatting
-      run: cmake --build build --target cmake-format-check
+      run: |
+        cmake --build build --target cmake-format-apply
+        echo "Errors found:"
+        git diff --exit-code
```

Comment:
`|| echo "no errors"` to leave no room for an empty line interpretation after the above echo.